lib/rx_nav/rx_norm.rb

```diff
@@ -104,6 +104,11 @@ def available_strength id
         property id, "available_strength"
       end
 
+      def displaynames
```

Comment:
Could you "rubyfy" this to `display_names`? Just a little more user friendly. Also would be safer to do something with some guards, e.g.

```ruby
def display_names
  query = "/displaynames"
  response = get_response_hash(query)
  response.try(:[], :display_terms_list).try(:[], :term)
end
```

which would not error if one of those keys didn't exist